src/scportrait/tools/parse/_parse_phenix.py

```diff
@@ -446,40 +460,44 @@ def replace_missing_images(self):
             )
 
     def define_copy_functions(self):
-        # define function for copying depending on if symlinks should be used or not
+        """Define function for copying depending on if symlinks should be used or not"""
         if self.export_symlinks:
-
             def copyfunction(input, output):
                 try:
-                    os.symlink(input, output)
+                    if platform.system() == 'Windows':
+                        # On Windows, use hard links when symlinks are requested
+                        if not os.path.exists(output):
```

Comment:
I think it would be good to add a warning here informing the user in the change in behaviour.